src/util/get-all-keys.js

```diff
@@ -0,0 +1,10 @@
+
+export default function keys(obj, enumerable = false) {
+  let listOfKeys = [
+    ...Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj),
```

Comment:
Thanks for the feedback! Yes, this was a leftover of a previous commit, since I was creating an array like ```[...array1, ...array2]```.
Now it doesn't makes much sense so I agree on just having 
```javascript 
const listOfKeys = Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj);
``` 

and afterwards, if it meets the condition, will push the symbols as @zzarcon suggest:
```javascript 
listOfKeys.push(...Object.getOwnPropertySymbols(obj))
```